doc/content/gateways/COTX-Gateways_index.md

```diff
@@ -0,0 +1,12 @@
+##COTX Gateways
+
+
+COTX product portfolio consists of IoT modules, LoRa gateways, node devices and all-in-one solutions. We have over 80 varieties of products in active production. 
+They offer high quality, outstanding radio performances and carrier-grade security, and support US915, AU915, AS923, EU868, IN865, RU867, CN470 etc. 
+Please consult https://www.cotxnetworks.com/ for the full product list.                                                                                                                                                                                                                                                                                                                     
+ 
+Find the documentation on how to connect:  
+#COTX X1 Lite Gateway   
+#COTX X1 Gateway   
+#COTX X3 Gateway   
+#COTX X5 Gateway   
```

Comment:
```suggestion
In the following subsection, you can find the documentation on how to connect COTX X1 Lite gateway.
```

Reply:
I see you have the intention to add more docs here, but you haven't added them yet. Please update this part when you add instructions for other gateways.